end-to-end-test/remote/specs/core/mutationMapperTool.spec.js

```diff
@@ -296,4 +296,108 @@ describe('Mutation Mapper Tool', function() {
             assert.ok(mutationCount.length > 0);
         });
     });
+
+    describe('GRCh38 example genomic changes input', () => {
+        beforeEach(() => {
+            var url = `${CBIOPORTAL_URL}/mutation_mapper`;
+            goToUrlAndSetLocalStorage(url);
+            browser.waitForVisible(
+                '[data-test=MutationMapperToolGRCh38Button]',
+                10000
+            );
+        });
+
+        it('should correctly annotate the genomic changes example with GRCh38 and display the results', () => {
+            // choose GRCh38
+            browser.click('[data-test=MutationMapperToolGRCh38Button]');
+            // the page will reloda after change genome build, wait until loading finished
+            browser.waitForVisible(
+                '[data-test=GenomicChangesExampleButton]',
+                10000
+            );
+            browser.click('[data-test=GenomicChangesExampleButton]');
+            browser.click('[data-test=MutationMapperToolVisualizeButton]');
+
+            waitForGenomeNexusAnnotation();
+
+            // mutations table should be visiable after oncokb icon shows up,
```

Comment:
```suggestion
            // mutations table should be visible after oncokb icon shows up,
```

Reply:
@onursumer good catch, I also fix this typo in other tests